packages/paste-core/primitives/box/src/StyleFunctions.ts

```diff
@@ -121,12 +122,11 @@ export const getCustomElementStyles = (props: StyledBoxProps): (() => CSSObject)
 
     if (themeElements[targetElement] != null) {
       const elementOverrides = themeElements[targetElement];
-      const computedStyles = css(elementOverrides)(props);
+      const computedStyles = css(elementOverrides)(props) as PasteCustomCSS;
       const {variants, ...elementStyles} = computedStyles;
       let variantStyles = {};
 
-      if (props.variant != null && variants != null) {
-        // @ts-ignore typing of css function returns a cssObject which doesn't think variants exists
+      if (props.variant != null && variants != null && variants[props.variant] != null) {
```

Comment:
Could you provide some context for this use case of the non-strict inequality operator?

Reply:
So this does a little more than it looks like, and it's also Typescript based.

`x != null` catches that `x !== undefined` _and_ `x !== null`, and when checking for those things this is a little safer this way. It's something we do through out the system, and @TheSisb can go into it more if you would like (he introduced it).

So, `props.variant`, and `variants` are optional types, which means they can be `undefined`, but also defined as `null`, so Typescript forces the check here

Reply:
+1 to what Simon said.  The only time I don't use strict equality (`===`) is when checking against null & undefined, via `!= null`